AcademicManagerV1/src/com/academic/model/Person.java

```diff
@@ -82,16 +81,16 @@ public boolean isMale() {
 		return sex;
```

Comment:
Κάτι ακόμα που τώρα το πρόσεξα. Στις isMale() και isFemale() πρέπει να  ελέγχεις την τιμή συγκεκριμένα, δηλαδή return this.sex == true για την isMale(), κ το αντίστοιχο για την isFemale.